src/styles.scss

```diff
@@ -0,0 +1,364 @@
+$yellowBlue: linear-gradient(45deg, rgba(64,59,74,1) 5%, rgba(224,228,138,1) 50%, rgba(64,59,74,1) 95%);
+$mojito: linear-gradient(45deg, rgba(29,151,108,1) 0%, rgba(147,249,185,1) 50%, rgba(29,151,108,1) 100%);
+$pink: linear-gradient(45deg, rgba(219,24,182,1) 0%, rgba(255,200,252,1) 50%, rgba(219,24,182,1) 100%);
+$purple: linear-gradient(45deg, rgba(65,39,173,1) 0%, rgba(213,154,238,1) 50%, rgba(65,39,173,1) 100%);
+$yellow: linear-gradient(45deg, rgba(255,252,0,1) 0%, rgba(255,255,255,1) 50%, rgba(255,252,0,1) 100%);
+$brown: linear-gradient(45deg, rgba(51,40,27,1) 0%, rgba(156,100,41,1) 50%, rgba(51,40,27,1) 100%);
+$orange: linear-gradient(45deg, rgba(241,39,17,1) 0%, rgba(245,175,25,1) 50%, rgba(241,39,17,1) 100%);
+$lightgray: linear-gradient(45deg, rgba(108,107,113,1) 0%, rgba(207,207,207,1) 50%, rgba(108,107,113,1) 100%);
+$darkgray: linear-gradient(45deg, rgba(31,28,44,1) 0%, rgba(146,141,171,1) 50%, rgba(31,28,44,1) 100%);
+$peachy: linear-gradient(45deg, rgba(65,94,249,1) 0%, rgba(248,71,110,1) 50%, rgba(65,94,249,1) 100%);
+$darkPurple: linear-gradient(45deg, rgba(51,0,27,1) 0%, rgba(255,0,132,1) 50%, rgba(51,0,27,1) 100%);
+$martini: linear-gradient(45deg, rgba(36,254,65,1) 0%, rgba(253,252,71,1) 50%, rgba(36,254,65,1) 100%);
+
+body{
+  display: flex;
+  justify-content: center;
+  flex-wrap: wrap;
+  font-size: 18px;
+  font-family: $font;
+  background-image: url($lightblueBg);
+  background-size: contain;
+
+  .flex{
+    display: flex;
+    justify-content: center;
+    flex-wrap: wrap;
+  }
+
+  .noclick{pointer-events: none;}
+
+  .msgDiv{
+    position: absolute;
+    align-self: center;
+    width: 100%;
+    height: 100%;
+    border-radius: 10px;
+    background-image: radial-gradient(circle, rgba(154,237,249,1) 35%, rgba(87,157,187,1) 100%);
+    z-index: 1;
+    transition: 1s;
+  }
+
+  .msgDiv p{
+    font-size: 20px;
+    display: flex;
+    justify-content: center;
+    flex-direction: column;
+    text-align: center;
+    height: 100%;
+    margin: 0;
+  }
+
+  .msgButton{
+    position: absolute;
+    bottom: 0px;
+    right: 0px;
+    cursor: pointer;
+    font-size: 17px;
+    font-family: $font;
+    background-image: $brownButtonColor;
+    color: black;
+    margin: 10px;
+    border: 2px solid black;
+    border-radius: 10px;
+    outline: none;
+  }
+
+  .msgButton:hover{
+    box-shadow: 0px 0px 12px 5px rgba(0,0,0,0.5);
+    transition: 0.1s;
+  }
+
+  .msgButton:active{
+    background-image: $brownButtonColorActive;
+  }
+
+  .blurredSpan{
+    border-radius: 10px;
+    backdrop-filter: blur(1px);
+    height: 26px;
+    margin: 5px 5px 10px;
+  }
+
+  .gameField{
+    display: grid;
+    justify-items: center;
+    align-items: center;
+    grid-gap: 5px;
+    padding: 6px;
+    margin: 10px;
+    border: 3px solid black;
+    border-radius: 10px;
+    backdrop-filter: blur(1px);
+    box-shadow: 0px 0px 20px 0px rgba(0,0,0,1);
+  }
+
+  .gameField--4x4 {
+    grid-template-columns: repeat(4, 1fr);
```

Comment:
here is a good place for parent selector too https://sass-lang.com/documentation/style-rules/parent-selector

Reply:
pls do similar fix in the whole file.
In the huge project better to split styles into several files e.g. field, card, button.

Reply:
Done